Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIPortCocoa.mm

```diff
@@ -205,16 +205,21 @@
     return *m_onDisconnect;
 }
 
-void WebExtensionContextProxy::dispatchPortMessageEvent(WebExtensionPortChannelIdentifier channelIdentifier, const String& messageJSON)
+void WebExtensionContextProxy::dispatchPortMessageEvent(WebExtensionPortIdentifier senderIdentifier, WebExtensionPortChannelIdentifier channelIdentifier, const String& messageJSON)
 {
     auto ports = WebExtensionAPIPort::get(channelIdentifier);
     if (ports.isEmpty())
         return;
 
     id message = parseJSON(messageJSON, { JSONOptions::FragmentsAllowed });
 
-    for (auto& port : ports)
+    for (Ref port : ports) {
```

Comment:
is `Ref` preferred in for loops? i thought it didn't matter in this case

Reply:
Yes, that is the new guidance use `Ref` or `RefPtr` instead of `auto` for refcounted types.